cmd/ff-proxy/main.go

```diff
@@ -629,7 +632,7 @@ func main() {
 				protocol = "https"
 			}
 
-			heartbeat(ctx, ticker.C, fmt.Sprintf("%s://localhost:%d", protocol, port), logger)
+			health.Heartbeat(ctx, ticker.C, fmt.Sprintf("%s://localhost:%d", protocol, port), logger)
```

Comment:
Should this always be localhost?

Reply:
I think so because we're pinging our own endpoint